var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -347,7 +359,7 @@ def patch_postdeps(self):
                     arg for arg in m.group(1).split(' ') if arg != '-l'),
                 'libtool')
 
-    @run_after('configure')
+    @run_after('cmake')
     def patch_libtool(self):
```

Comment:
The same here: no need to patch the `libtool` script (especially since it's not generated at all, which leads to a failure).